config/samples/ccruntime/peer-pods/kustomization.yaml

```diff
@@ -18,5 +18,11 @@ patches:
     - op: replace
       path: /spec/config/runtimeClassNames
       value: ["kata-remote"]
+    - op: add
+      path: /spec/config/defaultRuntimeClassName
+      value: "kata-remote"
```

Comment:
I can simply drop it, and avoid the confusion entirely.

Would that work better for you?

Reply:
I just dropped, so we don't have any change in the current behaviour

Reply:
I'm just wondering if others have opinions on us using the default runtimeclass name as I can imagine that someone like me might associate `kata` with `kata-qemu` and then in this set-up think they are creating a local hypervisor, rather than peer pod instance? It's probably a wider kata doc/UX think to think about.

Reply:
With what I kept here, at least for this PR, is following exactly what was done as part of the operator till Today.
For kata-containers x86_64 payload, the default is QEMU (kata-qemu).
For kata-containers s390x payload, the default is QEMU (kata-qemu)
The case for the remote hypervisor was the only one I was not sure on how to proceed.

In any case, I think that leaving the default as it's always been is the way to go, as long as we give the opportunity to the users to adapt it whatever is their preference.